### PR TITLE
Force the bundle major version to be over 3000

### LINKED ISSUE #12816: Can't install Terminal 1.12.10732.0 using DISM on Windows 11
### Windows Terminal version

1.12.10732.0

### Windows build number

10.0.22000.556

### Other Software

_No response_

### Steps to reproduce

```
SET WINTERMVERSION=1.12.10732.0
curl -Lo terminal-preinstall.zip https://github.com/microsoft/terminal/releases/download/v%WINTERMVERSION%/Microsoft.WindowsTerminal_Win10_%WINTERMVERSION%_8wekyb3d8bbwe.msixbundle_Windows10_PreinstallKit.zip
MKDIR term
tar -C term -xvf terminal-preinstall.zip
FOR %%i IN (term\*.msixbundle) DO SET MSIX=%%i
FOR %%i IN (term\*_License1.xml) DO SET LIC=%%i
DISM /Online /Add-ProvisionedAppxPackage /PackagePath:%MSIX% /LicensePath:%LIC%
```

### Expected Behavior

Installs the updated version, as it did with `WINTERMVERSION=1.11.3471.0` (without the extra `_Win10_` before the version in the download filename)

### Actual Behavior

```
2022-04-03 20:07:28, Error                 DISM   DISM Appx Provider: PID=1452 TID=7716 onecore\admin\appmodel\utilities\provisionhelper\msixprovisioningrequest.cpp(878)\AppxProvider.dll!00007FFC98D761A4: (caller: 00007FFC98D74734) ReturnHr(1) tid(1e24) C1570118     Msg:[Failed while adding package Microsoft.WindowsTerminal_1.12.10732.0_neutral_~_8wekyb3d8bbwe since an equal or higher version of the package, Microsoft.WindowsTerminal_2021.226.1915.0_neutral_~_8wekyb3d8bbwe, already exists] 
 - CAppxManager::ResultLoggingCallback
2022-04-03 20:07:28, Error                 DISM   DISM Appx Provider: PID=1452 TID=7716 onecore\admin\appmodel\utilities\provisionhelper\msixprovisioningrequest.cpp(180)\AppxProvider.dll!00007FFC98D74790: (caller: 00007FFC98D73F54) ReturnHr(2) tid(1e24) C1570118     Msg:[Failed to remove existing package version for 'Microsoft.WindowsTerminal_1.12.10732.0_neutral_~_8wekyb3d8bbwe'.] 
 - CAppxManager::ResultLoggingCallback
2022-04-03 20:07:28, Error                 DISM   DISM Appx Provider: PID=1452 TID=7716 onecore\admin\appmodel\utilities\provisionhelper\msixprovisioningrequest.cpp(79)\AppxProvider.dll!00007FFC98D743F4: (caller: 00007FFC98D7FF70) ReturnHr(3) tid(1e24) C1570118     Msg:[Failed to add main bundle package for provisioning.] 
 - CAppxManager::ResultLoggingCallback
2022-04-03 20:07:28, Error                 DISM   DISM Appx Provider: PID=1452 TID=7716 onecore\admin\appmodel\utilities\provisionhelper\msixpackageadapter.cpp(1439)\AppxProvider.dll!00007FFC98D7FFA0: (caller: 00007FFC98D75A10) ReturnHr(4) tid(1e24) C1570118     Msg:[Failed to initialize package adapter.] 
 - CAppxManager::ResultLoggingCallback
2022-04-03 20:07:28, Error                 DISM   DISM Appx Provider: PID=1452 TID=7716 onecore\admin\appmodel\utilities\provisionhelper\msixprovisioningrequest.cpp(651)\AppxProvider.dll!00007FFC98D75B54: (caller: 00007FFC98D7A0B4) ReturnHr(5) tid(1e24) C1570118     Msg:[Failed to create adapter for 'term\a372015b7dfd43638e7e21f052c12151.msixbundle'.] 
 - CAppxManager::ResultLoggingCallback
2022-04-03 20:07:28, Error                 DISM   DISM Appx Provider: PID=1452 TID=7716 onecore\admin\appmodel\utilities\provisionhelper\msixprovisioningmanager.cpp(920)\AppxProvider.dll!00007FFC98D7A0E8: (caller: 00007FFC98D4A2B8) ReturnHr(6) tid(1e24) C1570118     Msg:[Failed to add package 'term\a372015b7dfd43638e7e21f052c12151.msixbundle' to adapter set for processing.] 
 - CAppxManager::ResultLoggingCallback
```

### PR DESCRIPTION
**Pull request by DHowett**
Before #12691, the msixbundle version was of the format YYYY.MM.DD.0.
After #12691, it became the same as the build of Terminal it contained.

This caused some trouble for _some_ systems: major version 1 is much,
much smaller than 2022.

Adding 3000 to the major version component, _only for the bundle_, gets
around this. Ugh.

Fixes #12816.

### CODE REVIEW

File: build/pipelines/release.yml [hunk 1]
```diff
@@ -138,38 +138,38 @@ jobs:
...
-  - task: PowerShell@2
-    displayName: Move Loc files one level up
-    inputs:
-      targetType: inline
-      script: >-
-        $Files = Get-ChildItem . -R -Filter 'Resources.resw' | ? FullName -Like '*en-US\*\Resources.resw'
-
-        $Files | % { Move-Item -Verbose $_.Directory $_.Directory.Parent.Parent -EA:Ignore }
-      pwsh: true
+        #  - task: TouchdownBuildTask@1
```
Comment (DHowett) [inline build/pipelines/release.yml:141]: NFCI commit, will be reverted

**Comment by DHowett** [comment]
image

**Review by zadjii-msft — APPROVED** [review]
> There is a risk that using this version number will fail because it is
> thousands of numbers less than "2022". We'll have to see if the store
> rolls it out properly


Narrator: _it did not_

**Review by miniksa — APPROVED** [review]
Gross.

**Comment by DHowett** [comment]
@msftbot merge this in 1 minute

**Pull request merged**